proxy/http/HttpConnectionCount.cc

```diff
@@ -38,17 +39,27 @@ OutboundConnTrack::Imp OutboundConnTrack::_imp;
 OutboundConnTrack::GlobalConfig *OutboundConnTrack::_global_config{nullptr};
 
 const MgmtConverter OutboundConnTrack::MAX_CONV(
-  [](const void *data) -> MgmtInt { return static_cast<MgmtInt>(*static_cast<const decltype(TxnConfig::max) *>(data)); },
-  [](void *data, MgmtInt i) -> void { *static_cast<decltype(TxnConfig::max) *>(data) = static_cast<decltype(TxnConfig::max)>(i); });
+  [](const TSHttpTxn txnp, const void *data) -> MgmtInt {
```

Comment:
The signature should be `const HttpSM*, const void * data)`. Drop the name of the unused parameter.